pyproject.toml

```diff
@@ -101,6 +101,7 @@ scripts_are_modules = true
 warn_unused_ignores = true
 warn_redundant_casts = true
 no_warn_no_return = true
+show_traceback = true
```

Comment:
>  Isn't it useful to have it show the traceback for debugging any errors that appear?

Yes, exactly, that is what this change is doing. It is not necessary for the fix, but for the future it will help to faster pinpoint where problems come from.

Reply:
Ah, never mind, I read it the other way around (like if you were removing it). Makes sense now.